lib/ace/css/editor.css

```diff
@@ -200,31 +194,33 @@
     text-align: right;
     pointer-events: auto;
     height: 1000000px;
-    will-change: transform;
-    backface-visibility: hidden;
-    contain: strict;
+    contain: style size layout;
 }
 
 .ace_text-layer {
     font: inherit !important;
     position: absolute;
     height: 1000000px;
     width: 1000000px;
-    will-change: transform;
-    backface-visibility: hidden;
-    contain: strict;
+    contain: style size layout;
 }
 
-.ace_text-layer div {
-    contain: strict;
+.ace_text-layer > .ace_line, .ace_text-layer > .ace_line_group {
+    contain: style size layout;
     position: absolute;
     top: 0;
     left: 0;
     right: 0;
 }
 
-.ace_text-layer .ace_line_group .ace_line {
-    position: relative;
+ @media (-webkit-min-device-pixel-ratio: 1.5), (min-resolution: 1.5dppx) {
```

Comment:
Does this mean we only get hw acceleration on retina screens?

Reply:
yes, do we need to enable it on mac even if screen is not retina?